skimage/filters/_identity.py

```diff
@@ -0,0 +1,29 @@
+"""
+Identify function.
+
+Return original image.
+"""
+
+
+def identity(image, **kwargs):
```

Comment:
OK, I'm not yet familiar with the conventions which additions/functions are justified. Since I find the identity function (ignoring keyword arguments) really helpful for automated plotting of gallery examples I'll wait a few days to see if there are any supporters for this. Otherwise I'll remove it, alright?